pkg/tfgen/testdata/TestConvertViaPulumiCLI/GenerateSchema/Description.golden

```diff
@@ -85,7 +85,7 @@ public class App {
     public static void stack(Context ctx) {
         var aResource = new Resource("aResource", ResourceArgs.builder()
             .renamedInput1("hello")
-            .inputTwo(true)
+            .inputTwo("true")
```

Comment:
Hmmm this diff is interesting. Looks like javagen may have chosen a different default in absense of schema.